src/useSubscription.ts

```diff
@@ -2,13 +2,22 @@ import { useEffect } from 'react';
 import { GraphQLSubscriptionConfig, requestSubscription, OperationType } from 'relay-runtime';
 import { useRelayEnvironment } from './useRelayEnvironment';
 
+type SubscriptionConfig = {
+  skip?: boolean;
+}
+
 export function useSubscription<TSubscriptionPayload extends OperationType = OperationType>(
     config: GraphQLSubscriptionConfig<TSubscriptionPayload>,
+    opts?: SubscriptionConfig
 ): void {
     const environment = useRelayEnvironment();
+    const skip = opts?.skip ?? false
```

Comment:
Here I would prefer to declare the constant like this: **const skip = opts && opts.skip;** 
As I find it simple and it reduces the bundle size :)